src/server/debug.rs

```diff
@@ -142,12 +142,17 @@ impl<ER: RaftEngine> Debugger<ER> {
         let start_key = keys::REGION_META_MIN_KEY;
         let end_key = keys::REGION_META_MAX_KEY;
         let mut regions = Vec::with_capacity(128);
-        box_try!(db.scan_cf(cf, start_key, end_key, false, |key, _| {
+        box_try!(db.scan_cf(cf, start_key, end_key, false, |key, value| {
             let (id, suffix) = box_try!(keys::decode_region_meta_key(key));
-            if suffix != keys::REGION_STATE_SUFFIX {
-                return Ok(true);
+            if suffix == keys::REGION_STATE_SUFFIX {
+                let mut region_state = RegionLocalState::default();
+                region_state.merge_from_bytes(value)?;
+                if region_state.get_state() == PeerState::Tombstone {
```

Comment:
Then how to know how many tombstone states in the DB?

Reply:
Currently, it's impossible to know whether a region returned from this API is tombstoned. So it brings confusion. When do you need to know how many tombstone states in the DB?

Reply:
I believe you can use `tikv-ctl raft region -r id` to know if a region is tombstoned. I don't have a practical case for now, but it should be useful when it needs to analyze historical information.

Reply:
Reasonable. But just to mention, `tikv-ctl raft region -r id` doesn't show the peer state so far.